src/pages/MyTimePage/MyTimePage.tsx

```diff
@@ -177,13 +177,13 @@ export const MyTimePage = () => {
         schedulerInput.goals.push(obj);
       });
       console.log("input", schedulerInput);
-      const schedulerOutput: TaskItem[] = schedule(schedulerInput);
+      const schedulerOutput: TaskItem[] = (schedule(schedulerInput)).scheduled;
+      console.log("output", schedulerOutput);
```

Comment:
While we're here what do you think of console logging `impossible` tasks also?

Reply:
done